shared/src/balancer/pool_fetching.rs

```diff
@@ -14,6 +14,7 @@ use ethcontract::{BlockId, Bytes, H160, H256, U256};
 pub struct PoolTokenState {
     pub balance: U256,
     pub weight: U256,
+    pub scaling_factor: u8,
```

Comment:
In the contract code the scaling factor is `10^(18-decimals)`, so maybe we should call this differently, for example `scaling_exp`?

Reply:
Yes, good idea. I will change it. I am assuming you can handle the value then as a u8 and not as the actual contract value (like 10^x)